frontend/src/pages/modelRegistry/screens/ModelRegistrySelector.tsx

```diff
@@ -44,9 +49,12 @@ const ModelRegistrySelector: React.FC<ModelRegistrySelectorProps> = ({
         <SelectOption
           id={mr.metadata.name}
           key={mr.metadata.name}
-          value={getDisplayNameFromK8sResource(mr)}
+          value={getResourceNameFromK8sResource(mr)}
+          description={<Truncate content={getDescriptionFromK8sResource(mr)} />}
```

Comment:
Ok, this is interesting. So there are two things about the design that this doesn't match:
1. The tooltip has a bold header "<registry name> description" above its full content
2. The tooltip appears to be positioned relative to the entire SelectOption, not just the description, and positioned to the right.

In theory we could fix this by not using the `<Truncate>` component and instead using just the CSS styles for truncating the displayed text without adding a tooltip ([like in this HTML example](https://www.patternfly.org/components/truncate/html#default), except with the classNames imported from `@patternfly/react-styles`), and then add our own custom tooltip around the SelectOption. However I messed with that a little bit, and if you wrap SelectOption in Tooltip it causes an error because there is special logic in SelectGroup that expects SelectOption to be a direct child. I tinkered with other ways to position the tooltip where we want it and gave up.

The main problem here is that we're still using the deprecated Select component. If we were using the new PF5 Select, SelectOption would be inheriting props from MenuItem and we'd be able to pass `tooltipProps` to it to get our custom tooltip with the right positioning and the header text.

So I see two options here:
1. Upgrade this whole component to the new Select as part of this PR and use the above solution. @gitdallas I'll let you decide if that's practical to do quickly.
2. Do a half-measure for now (maybe just pass `tooltipPosition="right"` to Truncate so at least that is positioned a bit better when you hover the description text) and either open a followup tech debt issue to improve this later or let it go.

@yih-wang are you ok with option (2) above for MVP? We'd have a tooltip that only appears when you hover the small description text, and it won't have the bold "<registry name> description" header text in it. If so, should we open an issue to make it match the mockup exactly after MVP or is that fine to keep?